Clarification of hover implementation (#2)
DrDiman/CSS-Bayan-task: README.md

6. Responsive design with three breakpoints for mobile, tablet, and desktop exist. Accordion is displayed correctly at `mobile 320x568, tablet 820x1180, desktop 1920×1080` **+10**
7. All visual effects when the cursor is hovering over the memes, when the mouse is down on a meme, and when a meme is selected are implemented **+10**
8. The entire row (text, icon, and meme image) clickable **+5**

> 9. Mobile first approach is used - cursor over the memes (hover) effect exists only for desktop devices. Hover effect should be implemented at `1920x1080`, and should not exist at `320x568`. Hover implementation between these resolutions are up to the developer. **+10**

Reviewer: Cursor over the memes (hover) effect exists only for desktop devices. Hover effect should be implemented at `1920x1080`, and should not exist at `320x568` and at 820x1180. **+10**

Предполагалось, что и на планшете не будет никаких ховеров.

Reviewer 2: Ховер должен быть четко привязан к разрешению независимо от типа устройства? 
Не делать ховер для тач девайсов (но добавить его для девайсов с курсором на любых разрешениях) в данном случае будет неверной реализацией?

Reviewer 3: Получается планшет это от 820 до 1920? А десктоп это 1920+? Мне кажется вот это условия надо помягче,  или уточнить что десктоп это строго 1920+ т.к. у многих в чате были вопросы по этому поводу. У меня на проектах к примеру десктоп 1024+, т.к. мониторы не у всех 1920+. Не совсем ясно как человек скажем с MacAir (1280px) должен проверять. Т.е у него десктоп, но ховеры не должны работать, чтобы они заработали, он должен знать как зайти в devtools и там настроить отображение под нужный brakepoint. Судя по вопросам в чате не все так смогут.

Reviewer 4: Вроде вот неплохой вариант: 

1. Проверяются 3 брейкпоинта:
    mobile 320x568, tablet 820x1180, desktop 1920×1080;

2. На mobile 320x568 и tablet 820x1180 не должно быть ховера

3. 1920x1080 обязательно должен быть ховер

Reviewer: Ховер может быть привязан к устройству, но при проверке должны проверяться три разрешения только. Думаю стоит этот флоу включить в форму кросс чека.

Reviewer: `Не делать ховер для тач девайсов (но добавить его для девайсов с курсором на любых разрешениях) в данном случае будет неверной реализацией?`

главное чтобы проверяющий смог проверить 3 разрешения, описанные в задании и ховеров не было на мобиле и теблете

Reviewer 5: Ховер **должен** быть привязан к устройству
Какая польза мне от ховера на тачёвом ноуте с разрешением 2560x1600 если он там не работает?
По моему это общее место во фронте использовать ховер только там где он поддерживается
Инструменты для этого есть
```css
@media (hover: hover) and (pointer: fine) {
	.accordion:hover .accordion_title {
		color: var(--color_dark);
	}
}
```

Reviewer: Все так. 
В реальности ховер должен быть только там, где он поддерживается.

В текущем задании лучше оставить три брейкпоинта, на которых необходимо проверить. Из практики прошлых заданий - люди спекулируют понятиями десктопа и мобилки и если не указать точных разрешений, то и делают разницу между разрешениями девайсов в 10px и мол ничего не нарушили..

Reviewer 5: Я всё равно этой идеи не понимаю зачем привязывать ховер к разрешению? Получается курсы учат студентов плохому. Как спекуляции людей могут повлиять на работу ховера? Он либо работает либо нет и это можно проверить в devtools хрома

Reviewer: Если оставим такую формулировку этого пункта:

`Cursor over the memes (hover) effect exists only for desktop devices.` убираем мобайл ферст.

А в форме кроссчека сделаем для этого пункта: 

```
- На мобайл версии (тач девайсе) не должно быть ховера

- на десктоп версии обязательно должен быть ховер.
```

Будут недопонимания/разночтения?

Reviewer 5: >Будут недопонимания/разночтения?

Думаю, да
Потому что выше есть упоминания про mobile и desktop и студент кмк мысленно привяжется к ним (desktop 1920×1080)
Хоть ipad и попадает под 1920×1080, но какой это desktop
![23070110057](https://user-images.githubusercontent.com/8797432/224473376-40d68c35-7e5f-432e-bf15-3e6ac6ab7bc8.png)

На mdn написано когда ховер используется https://developer.mozilla.org/en-US/docs/Web/CSS/@media/hover
![23070111348](https://user-images.githubusercontent.com/8797432/224473391-f02ebc9c-a3db-40eb-9925-b1f7e6f63530.png)

То есть формулировка должна быть вроде: Использование ховера допускается только на устройствах ввода поддерживающих ховер.

Reviewer 6: Этот вариант был самым прозрачным 
https://github.com/DrDiman/CSS-Bayan-task/pull/2#discussion_r1133045062

Reviewer: тогда оставляем такую формулировку этого пункта:

`Cursor over the memes (hover) effect only exists for devices that can support hover.` убираем мобайл ферст.

А в форме кроссчека сделаем для этого пункта:
```
- На тач девайсе (телефон, планшет) не должно быть ховера эффекта;
- На компьютере должны быть ховер эффекты.
```

думаю, что такой вариант ок.

Reviewer 7: По-моему самый логичный вариант сделать проверку ховера там где он доступен (открыли на пк или ноуте с маленьким разрешением, проверили есть ли ховер. Там же в девтулс в панеле устройств проверили, если нет ховера, значит всё окей) . А на брейкпойнтах для мобилок, планшетов и пк проверили адаптацию(размеры блоков, шрифтов и т.п.)

> 9. Mobile first approach is used - cursor over the memes (hover) effect exists only for desktop devices. Hover effect should be implemented at `1920x1080`, and should not exist at `320x568`. Hover implementation between these resolutions are up to the developer. **+10**

Reviewer: ```suggestion
9. Cursor over the memes (hover) effect only exists for devices that can support hover. **+10**
```
